README.md

```diff
@@ -25,6 +25,19 @@ IPFS is a global, versioned, peer-to-peer filesystem. It combines good ideas fro
 
 For more info see: https://docs.ipfs.tech/concepts/what-is-ipfs/
 
+# ZSH
+
+The zsh shell is also supported:
+
+The simplest way to "eval" the completions logic:
+
+```bash
+> eval "$(ipfs commands completion zsh)"
+```
+
+To install the completions permanently, they can be moved to
+`/etc/bash_completion.d` or sourced from your `~/.zshrc` file.
+
```

Comment:
Please move this to [`docs/command-completion.md`](https://github.com/ipfs/kubo/blob/master/docs/command-completion.md)